### PR TITLE
wip feat(cxl-ui): cxl-app-layout slot name=action-bar

### PR DESCRIPTION
**Pull request by lkraav**

### CODE REVIEW

**Comment by anoblet** [comment]
The UI looks fantastic!

I'm a little confused though as to why we aren't wrapping these features into their own components? If we did something like:

```
<cxl-action-bar lesson-id=${...}></cxl-action-bar>
```

Pass in the necessary information from WP and use methods on the component like `completeLesson` to hit the API and update state when the promise resolves?

There's probably something I'm missing here :)

**Comment by lkraav** [comment]
> The UI looks fantastic!
> 
> I'm a little confused though as to why we aren't wrapping these features into their own components? If we did something like:
> 
> ```
> <cxl-action-bar lesson-id=${...}></cxl-action-bar>
> ```
> 
> Pass in the necessary information from WP and use methods on the component like `completeLesson` to hit the API and update state when the promise resolves?
> 
> There's probably something I'm missing here :)

I'm open to good refactor ideas. Was looking to test some simple thing, first.

File: packages/storybook/cxl-ui/cxl-app-layout/layout=2c-r.stories.js [hunk 1]
```diff
@@ -370,6 +366,49 @@ export const CXLAppLayout2cr = () => {
           <p>Ending paragraph here.</p>
         </div>
       </article>
+
+      <div slot="action-bar">
+        <!--
```
Comment (anoblet) [inline packages/storybook/cxl-ui/cxl-app-layout/layout=2c-r.stories.js:371]: Remove commented code?

**Comment by anoblet** [comment]
I made a POC of separating the action bar into it's own component. It can be seen here:

https://github.com/anoblet/aybolit/pull/18

**Comment by pawelkmpt** [comment]
Closed in favor of #189

**Pull request closed without merging**